src/main/java/com/employeesService/EmployeesService/controller/DepartmentController.java

```diff
@@ -0,0 +1,48 @@
+package com.employeesService.EmployeesService.controller;
+
+import com.employeesService.EmployeesService.model.Department;
+import com.employeesService.EmployeesService.service.DepartmentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/departments")
+public class DepartmentController {
+
+    private DepartmentService departmentService;
+
+    @Autowired
+    public DepartmentController(DepartmentService departmentService){
+        this.departmentService=departmentService;
+    }
+
+
```

Comment:
One space should be between methods, and ALWAYS format before committing new code